From d8fcd6b7ce382fcbe7c36237a3e7910d09f7bf92 Mon Sep 17 00:00:00 2001
From: Max Bires <jbires@google.com>
Date: Mon, 24 Apr 2017 19:56:38 -0700
Subject: [PATCH 2/2] sepolicy: qcom: Fixing camera app not launching

denied { read } for pid=9669 comm="id.GoogleCamera"
name="u:object_r:camera_prop:s0" dev="tmpfs" ino=17770
scontext=u:r:untrusted_app:s0:c512,c768
tcontext=u:object_r:camera_prop:s0 tclass=file

Bug: 37633957
Test: Camera launches
Change-Id: I2b68a13eaea0ee83e83f8e92cfda9d46c4531060
---
 qcom/common/app.te | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 qcom/common/app.te

diff --git a/qcom/common/app.te b/qcom/common/app.te
new file mode 100644
index 0000000..036cccd
--- /dev/null
+++ b/qcom/common/app.te
@@ -0,0 +1,2 @@
+# For the camera app
+get_prop(appdomain, camera_prop)
-- 
2.7.4

